pkg/util/workloads_utils.go

```diff
@@ -212,6 +226,95 @@ func GenRandomStr(length int) string {
 	return rand.SafeEncodeString(randStr)
 }
 
+func PatchUpdateStrategy(c client.Client, object client.Object, updateStrategy map[string]interface{}) error {
+	patch := map[string]interface{}{
+		"spec": map[string]interface{}{
+			"updateStrategy": updateStrategy,
+		},
+	}
+
+	patchByte, _ := json.Marshal(patch)
+	clone := object.DeepCopyObject().(client.Object)
+	return c.Patch(context.TODO(), clone, client.RawPatch(types.MergePatchType, patchByte))
+}
+
+func ReleaseWorkload(c client.Client, object client.Object) error {
+	_, found := object.GetAnnotations()[BatchReleaseControlAnnotation]
+	if !found {
+		klog.V(3).Infof("Workload(%v) is already released", client.ObjectKeyFromObject(object))
+		return nil
+	}
+
+	clone := object.DeepCopyObject().(client.Object)
+	patchByte := []byte(fmt.Sprintf(`{"metadata":{"annotations":{"%s":null}}}`, BatchReleaseControlAnnotation))
+	return c.Patch(context.TODO(), clone, client.RawPatch(types.MergePatchType, patchByte))
+}
+
+func ClaimWorkload(c client.Client, planController *v1alpha1.BatchRelease, object client.Object, patchUpdateStrategy map[string]interface{}) error {
+	if controlInfo, ok := object.GetAnnotations()[BatchReleaseControlAnnotation]; ok && controlInfo != "" {
+		ref := &metav1.OwnerReference{}
+		err := json.Unmarshal([]byte(controlInfo), ref)
+		if err == nil && ref.UID == planController.UID {
+			klog.V(3).Infof("Workload(%v) has been controlled by this BatchRelease(%v), no need to claim again",
```

Comment:
v(5) ?

Reply:
v(3) is appropriate. 
This function is only executed once during rollout.